playbooks/provisioner/openstack/main.yaml

```diff
@@ -4,3 +4,5 @@
   roles:
       - { role: openstack/neutron, action: setup }
```

Comment:
1. This isn't OVB :)
2. You are reviewing part that was already reviewed by you in 'openstack provisioner' patch. we can change it, but I don't think this is the place to do it.